test/cpp/interop/BUILD

```diff
@@ -228,7 +228,9 @@ grpc_cc_binary(
         "absl/flags:flag",
     ],
     deps = [
+        "//:grpcpp_admin",
```

Comment:
Buildifier will complain. Let's format it.

```
tools/distrib/buildifier_format_code.sh
```

Reply:
Done